src/Workspaces/CoreTest/Host/WorkspaceServices/TestOptionService.cs

```diff
@@ -1,71 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Immutable;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Options.Providers;
-using Microsoft.CodeAnalysis.Shared.Utilities;
-using Microsoft.CodeAnalysis.Test.Utilities;
-
-namespace Microsoft.CodeAnalysis.UnitTests
-{
-    internal static class TestOptionService
-    {
```

Comment:
Methods moved to src/Workspaces/CoreTest/WorkspaceServiceTests/GlobalOptionServiceTests.cs